ghwatch.py

```diff
@@ -310,19 +313,32 @@ def main():
 
     Key.start(hide_cursor=True)
 
-    t = 0
+    refr_t = 0
+    meta_t = 0
+    meta_i = config.get('auto-update', 0)
```

Comment:
Please call this variable `auto_update` or something else clearer (using `t` for the counter is fine).

Reply:
Done!! :D